webapp/src/cljs/lipas/ui/map/map.cljs

```diff
@@ -195,6 +195,7 @@
         popup-overlay (init-overlay)
 
         opts #js {:target   "map"
+                  :keyboardEventTarget js/document
```

Comment:
Didn't test this myself yet but just wondering if this captures keyboard events also within textfields when you move the cursor or in some other undesired circumstances?

Maybe an alternative is to bind the `:keyboardEventTarget` to this div instead? https://github.com/lipas-liikuntapaikat/lipas/blob/master/webapp/src/cljs/lipas/ui/map/map.cljs#L620-L623

Reply:
Tested and this works just like intended and didn't notice any undesired side-effects. :) nice job!